test/benchmark_test.js

```diff
@@ -1,18 +1,39 @@
-var cryptohat = require("..");
-var hat = require("hat");
+(function() {
+
+var cryptohat = __test.cryptohat;
+var hat = __test.hat;
 
 // Runs the target function for a number of iterations and returns the number
 // of nanoseconds passed per iteration.
-var benchmark = function (iterations, code) {
-  var t0 = process.hrtime();
-  for (var i = 0; i < iterations; ++i) {
-    code();
+var benchmark = null;
+
+if (typeof(process) !== "undefined" && typeof(process.hrtime) === "function") {
```

Comment:
I used the pattern that you suggested here:
https://github.com/heap/cryptohat/pull/6/files#diff-168726dbe96b3ce427e7fedce31bb0bcR93